src/App/Fossa/FossaAPIV1.hs

```diff
@@ -390,6 +394,27 @@ archiveBuildUpload apiOpts archiveProjects = runEmpty $
         req POST (archiveBuildURL baseUrl) (ReqBodyJson archiveProjects) bsResponse (baseOpts <> opts)
     pure (responseBody resp)
 
+---------- license-scan build queueing. This Endpoint ensures that after a license-scan is uploaded, it is scanned.
+
+licenseScanFinalizeUrl :: Url 'Https -> Url 'Https
+licenseScanFinalizeUrl baseUrl = baseUrl /: "api" /: "license_scan" /: "finalize"
+
+licenseScanFinalize ::
+  (Has (Lift IO) sig m, Has Diagnostics sig m) =>
+  ApiOpts ->
+  ArchiveComponents ->
+  m (Maybe C.ByteString)
```

Comment:
Why is the return type a bytestring?  Generally, we maintain that the API function should handle all data transformation and parsing from the server before returning that value to the caller.

Reply:
I see that the old archive upload function did that, let's leave a TODO to fix this later.

Reply:
Yeah, but let's make it a todo to keep this ticket smaller.